src/main/java/picard/fingerprint/ExtractFingerprint.java

```diff
@@ -67,6 +76,16 @@
             "If argument EXTRACT_CONTAMINATION=true the resulting samplename will be \"<SAMPLE>-contamination\" (if not provided).", optional = true)
     public String SAMPLE_ALIAS = null;
 
+    @Argument(doc = "SAM/BAM or VCF file of uncontaminated sample of the same individual whose information we are trying to remove.  If we are extracting the fingerprint of the contaminating sample," +
```

Comment:
I think that the use of "remove" and "extract" is very confusing here....this needs to be rethought and re-written, especially as there are currently two CLPs (ExtractFingerprint and IdentifyContaminant) that use 
this class.